.ci_scripts/run_cmake.sh

```diff
@@ -10,6 +10,9 @@ case "$BUILD_TARGET" in
 "mac")
 	mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=Release ..
 	;;
+"appimage")
+	mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr ..
```

Comment:
Do we need the `CMAKE_INSTALL_PREFIX` when we tell `make install` to use a different `DESTDIR`?

Reply:
`CMAKE_INSTALL_PREFIX` is the base directory for installing the game, which is prepended to `DESTDIR` which contains the path where we store the base directory and the children. I suppose it could be all moved to `DESTDIR` but it would be less idiomatic. See https://cmake.org/cmake/help/v3.16/envvar/DESTDIR.html

See https://cmake.org/cmake/help/v3.13/envvar/DESTDIR.html